cranelift/codegen/src/isa/aarch64/abi.rs

```diff
@@ -450,14 +447,14 @@ impl ABIBody for AArch64ABIBody {
     fn gen_copy_arg_to_reg(&self, idx: usize, into_reg: Writable<Reg>) -> Inst {
         match &self.sig.args[idx] {
             &ABIArg::Reg(r, ty) => Inst::gen_move(into_reg, r.to_reg(), ty),
-            &ABIArg::Stack(off, ty) => load_stack(off + 16, into_reg, ty),
+            &ABIArg::Stack(off, ty) => load_stack_from_fp(off + 16, into_reg, ty),
```

Comment:
I just realized: should this `16` be dependent on the calling convention? IIRC the Baldrdash/Wasm stack frame is actually 24 bytes, not 16 (and maybe would be padded to 32 to maintain 16-alignment, I'm not sure)?

Reply:
Yep, this is handled in #1530 :)